requirements/ansible-galaxy.yml

```diff
@@ -38,3 +38,18 @@
   name: onaio.gpg-import
   version: c5bb983d5b4bc5fec82a03d71543fefbe2ddd06e
   scm: git
+
+- src: https://github.com/onaio/ansible-java.git
+  name: onaio.ansible-java
```

Comment:
Don't think there's a need of adding this here as a dependency if we have also added it as a git submodule. I don't see an update on the .gitmodule files but I see a submodule added (roles/ansible-java) so that's another thing to check out.

Reply:
the submodule removed